src/test/java/org/bank/BankAccountManagerTest.java

```diff
@@ -0,0 +1,95 @@
+package org.bank;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BankAccountManagerTest {
+    private @InjectMocks BankAccountManager manager;
+    private @Mock BankAccountHelper helper;
+    private BankAccount fromAccount;
+    private BankAccount toAccount;
+
+    @Test
+    public void transferMoney_to() throws InvalidBankOperationException {
+        fromAccount = new BankAccount();
```

Comment:
If you always create suche beans, let's move creation to a separate method (probably, with needed parameters)